fixtures/american-dj/flat-par-qa12xs.json

```diff
@@ -0,0 +1,618 @@
+  "meta": {
+    "authors": ["TAKU"],
+    "createDate": "2018-08-05",
+    "lastModifyDate": "2018-08-05"
+  },
+  "manualURL": "http://intranet.americandj.com/ItemRelatedFiles/8308/flat_par_qa12xs.pdf",
+  "physical": {
+    "dimensions": [280, 115, 328],
+    "weight": 3.2,
+    "power": 80,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "RGBA QUAD LED"
+    },
+    "lens": {
+      "degreesMinMax": [25, 25]
+    },
+    "focus": {
+      "type": "Fixed"
+    }
+  },
+  "availableChannels": {
+    "Color Macros": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 15],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [16, 31],
+          "type": "ColorPreset",
+          "colors": ["#ff0000"],
+          "comment": "Red"
+        },
+        {
+          "dmxRange": [32, 47],
+          "type": "ColorPreset",
+          "colors": ["#00ff00"],
+          "comment": "Green"
+        },
+        {
+          "dmxRange": [48, 63],
+          "type": "ColorPreset",
+          "colors": ["#0000ff"],
+          "comment": "Blue"
+        },
+        {
+          "dmxRange": [64, 79],
+          "type": "ColorPreset",
+          "colors": ["#ffbf00"],
+          "comment": "Amber"
+        },
+        {
+          "dmxRange": [80, 95],
+          "type": "ColorPreset",
+          "colors": ["#ffff00"],
+          "comment": "Yellow"
+        },
+        {
+          "dmxRange": [96, 111],
+          "type": "ColorPreset",
+          "colors": ["#ff00ff"],
+          "comment": "Pink"
+        },
+        {
+          "dmxRange": [112, 127],
+          "type": "ColorPreset",
+          "colors": ["#ff6000"],
+          "comment": "Orange"
+        },
+        {
+          "dmxRange": [128, 143],
+          "type": "ColorPreset",
+          "colors": ["#00ffff"],
+          "comment": "Cyan"
+        },
+        {
+          "dmxRange": [144, 159],
+          "type": "ColorPreset",
+          "colors": ["#92ff00"],
+          "comment": "Lime"
+        },
+        {
+          "dmxRange": [160, 175],
+          "type": "ColorPreset",
+          "colors": ["#ffbfff"],
+          "comment": "Lavender"
+        },
+        {
+          "dmxRange": [176, 191],
+          "type": "ColorPreset",
+          "colors": ["#ffffff"],
+          "comment": "White"
+        },
+        {
+          "dmxRange": [192, 207],
+          "type": "ColorPreset",
+          "colors": ["#ffdf00"],
+          "comment": "Warm Yellow"
+        },
+        {
+          "dmxRange": [208, 223],
+          "type": "ColorPreset",
+          "colors": ["#ff6080"],
+          "comment": "Light Red"
+        },
+        {
+          "dmxRange": [224, 239],
+          "type": "ColorPreset",
+          "colors": ["#92ff92"],
+          "comment": "Light Green"
+        },
+        {
+          "dmxRange": [240, 255],
+          "type": "ColorPreset",
+          "colors": ["#ffdf80"],
+          "comment": "Red, Green, Blue, Amber"
```

Comment:
If you stick to the resulting color names instead of the individual components in these capabilities, then use "Light Amber" here please.